xpmem-kmod.spec

```diff
@@ -2,21 +2,26 @@
 #define buildforkernels current
 #define buildforkernels akmod
 
-%define kernel_release %(uname -r | sed -e 's/\.[^.]*$//g')
+%{!?kversion: %define kversion %(uname -r | sed -e 's/\.[^.]*$//g')}
+%{!?version: %define version 2.6.5}
 %global debug_package %{nil}
 
 Summary: XPMEM: Cross-partition memory
-Name: xpmem-kmod-%{kernel_release}
-Version: 2.6.5
+Name: xpmem-kmod-%{kversion}
+Version: %{version}
 Release: 0
 License: GPLv2
 Group: System Environment/Kernel
 Packager: Nathan Hjelm
 Source: xpmem-0.2.tar.bz2
-BuildRoot: %{_tmppath}/%{name}-0.2-build
-Requires: kernel = %{kernel_release}
+BuildRoot: %{_tmppath}/%{name}-%{version}-build
+Requires: kernel >= %{kversion}
```

Comment:
why not = ?

Reply:
Because with weak-updates the module works for new compatible versions without recompiling. (e.g. a module for 4.18.0-305.3.1.el8_4 also works with 4.18.0-305.19.1.el8_4)